c4p2n/api.py

```diff
@@ -28,15 +30,27 @@ def health_check() -> Dict[str, str]:
 
 
 @app.post("/call_for_paper", dependencies=[Depends(signature_header_security)])
-def call_for_paper_webhook(request: CallForPaperRequest,) -> Dict[str, Any]:
+async def call_for_paper_webhook(request: CallForPaperRequest,) -> Dict[str, Any]:
     try:
         prepared_request = request.prepare()
     except ValidationError:
+        await telegram_api.triggers_api.typeform_invalid()
```

Comment:
Here we can use `asyncio.create_task` instead of a bare await as we can send trigger in the background and there is no need to wait till it's sent to return an HTTP request.

Reply:
Let's use not `asyncio.create_task`, but https://fastapi.tiangolo.com/tutorial/background-tasks/

Reply:
Yep, if FastAPI provides a special way for working with background tasks let's use it.